src/settings/MappingProfiles/detailsSections/edit/MappingInstanceDetails.js

```diff
@@ -101,3 +108,10 @@ export const MappingInstanceDetails = ({
     </AccordionSet>
   );
 };
+
+MappingInstanceDetails.propTypes = {
+  initialFields: PropTypes.shape(mappingInstanceInitialFieldsShape).isRequired,
+  referenceTables: PropTypes.shape(mappingInstanceRefTablesShape).isRequired,
+  setReferenceTables: PropTypes.func.isRequired,
+  okapi: PropTypes.shape(okapiShape).isRequired,
```

Comment:
What do you think about refactoring to be able to use it in the following way? As I see this how we use `stripesShape`.
```suggestion
  okapi: okapiShape.isRequired,
```

Reply:
Good idea, thanks!
Done